imageroot/actions/configure-module/10validate

```diff
@@ -65,7 +65,7 @@ if request["dhcp-server"]["enabled"]:
         sys.exit(2)
 
 if request["dns-server"]["enabled"]:
-    is_dns_bound = network.are_ports_53_bound()
+    is_dns_bound = network.are_ports_53_bound() or bool(network.get_local_samba_dcs())
```

Comment:
Maybe is better to add a `network.get_local_samba_dcs()` check inside the `are_ports_53_bound`, the result should be the same... right?